src/widgets/crud-data-table/experimental_crud-data-table.tsx

```diff
@@ -126,10 +127,6 @@ function CrudDataTableInternal<TData extends DataTableRow<TData>, TValue>(
     ? () => dialogRef.current?.open()
     : undefined;
 
-  const handleOnRowClick = onUpdateSubmit
-    ? (row: TData) => dialogRef.current?.open(row)
-    : undefined;
-
```

Comment:
Don't delete the old code. Instead, do the following:
```
let handleOnRowClick = onRowClick;
if (handleOnRowClick === undefined && onUpdateSubmit) {
  handleOnRowClick = (row: TData) => dialogRef.current?.open(row)
}
```
This way the old logic still works, but if we supply onRowClick, it overrides opening the handler.